### PR TITLE
Feature/html structure for memory game

### PR DESCRIPTION
**Pull request by magdalena-dering**
Hi, 

I created basic html structure for memory game. I added some oldschool style.
Review it, in ur free time.
My next goal is make, memory game responsive view.

Thank u a lot ;)

### CODE REVIEW

**Comment by bachbach** [comment]
Help wanted? What do you mean?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                <br> time 10 sec. - 20 sec. - 5p.
+                <br> time > 20 sec - 1p.</p>
+        </p>
+        <div class="marks">
+            <div class="attempts">
+                <h2 class="inp_title">attempts</h2>
+                <input type="text" name="attempts" class="inp_attempts">
+            </div>
+            <div class="points">
+                <h2 class="inp_title">points</h2>
```
Comment (bachbach) [inline index.html]: what means `inp` prefix?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                <input type="text" name="points" class="inp_points">
+            </div>
+        </div>
+        <button class="restart_btn">restart</button>
+    </header>
+    <div class="container">
+        <div id="game">
+            <div class="col">
+                <div class="game_squer"></div>
```
Comment (bachbach) [inline index.html]: square :)

File: index.html (hunk 1 above)
Comment (magdalena-dering) [reply index.html]: inp = input? should I change it?

File: index.html (hunk 2 above)
Comment (magdalena-dering) [reply index.html]: ;D thanks

File: index.html (hunk 1 above)
Comment (bachbach) [reply index.html]: Yes.

**Pull request merged**